server/data/communityApiClient.ts

```diff
@@ -5,19 +5,25 @@ import {
   CommunityApiManagedOffender,
   CommunityApiTeamManagedCase,
 } from '../@types/communityClientTypes'
-import { User } from '../@types/CvlUserDetails'
 
 export default class CommunityApiClient extends RestClient {
   constructor() {
     super('Community API', config.apis.communityApi as ApiConfig)
   }
 
-  async getStaffDetail(user: User): Promise<CommunityApiStaffDetails> {
+  async getStaffDetailByUsername(username: string): Promise<CommunityApiStaffDetails> {
     return (await this.get({
-      path: `/secure/staff/username/${user.username}`,
+      path: `/secure/staff/username/${username}`,
     })) as Promise<CommunityApiStaffDetails>
   }
 
+  async getStaffDetailsByUsernameList(usernames: string[]): Promise<CommunityApiStaffDetails[]> {
```

Comment:
Endpoint to get staff details by a list of comUsernames. Used to load COM column on OMU caseload